drivers/scheduler/rke/rke.go

```diff
@@ -1,28 +1,270 @@
+	//TODO: Get the values from config map after https://github.com/portworx/torpedo/pull/1517 is merged
+	username := "admin"
+	password := "1JVWD8juKPB8fVgK"
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	client := &http.Client{Transport: tr}
+	body := strings.NewReader(fmt.Sprintf(`{"username":"%s","password":"%s"}`, username, password))
+	req, err := http.NewRequest("POST", rancherURL, body)
+	if err != nil {
+		log.Errorf("Failed to create rancher POST request:", err)
+		return &rkeParameters, err
+	}
+	req.Header.Add("Content-Type", "application/json")
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Infof("Failed to send request:", err)
+		return &rkeParameters, err
+	}
+	defer resp.Body.Close()
+	respBody, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Infof("Failed to read response:", err)
+		return &rkeParameters, err
+	}
+	response := string(respBody)
+	err = json.Unmarshal([]byte(response), &rkeData)
+	if err != nil {
+		log.Infof("Error:", err)
+		return &rkeParameters, err
+	}
+	rkeParameters.Endpoint = endpoint
+	rkeToken := rkeData["token"].(string)
+	if len(strings.Split(rkeToken, ":")) != 2 {
+		return &rkeParameters, fmt.Errorf("invalid RKE token value")
+	}
+	rkeParameters.Token = rkeToken
+	rkeParameters.AccessKey = strings.Split(rkeToken, ":")[0]
+	rkeParameters.SecretKey = strings.Split(rkeToken, ":")[1]
+	return &rkeParameters, err
+}
+
+// GetActiveRancherClusterID returns the ID of active rancher cluster
+func (r *Rancher) GetActiveRancherClusterID(clusterName string) (string, error) {
+	var clusterId string
+	clusterCollection, err := r.client.Cluster.List(nil)
+	if err != nil {
+		return "", err
+	}
+	for _, cluster := range clusterCollection.Data {
+		if cluster.Name == clusterName {
+			clusterId = cluster.ID
+		}
+	}
+	return clusterId, nil
+}
+
+// CreateRancherProject creates new project in rancher cluster
+func (r *Rancher) CreateRancherProject(projectName string, projectDescription string) (*rancherClient.Project, error) {
+	var clusterId string
+	var newProject *rancherClient.Project
+	clusterId, err := r.GetActiveRancherClusterID("local")
+	if err != nil {
+		return newProject, err
+	}
+	projectRequest := &rancherClient.Project{
+		Name:        projectName,
+		Description: projectDescription,
+		ClusterID:   clusterId,
+	}
+	newProject, err = r.client.Project.Create(projectRequest)
+	if err != nil {
+		return newProject, err
+	}
+	return newProject, nil
+}
+
+// GetProjectID return the project ID
+func (r *Rancher) GetProjectID(projectName string) (string, error) {
+	var projectId string
+	projectList, err := r.client.Project.List(nil)
+	if err != nil {
+		return projectId, err
+	}
+	for _, project := range projectList.Data {
+		if project.Name == projectName {
+			projectId = project.ID
+			return projectId, nil
+		}
+	}
+	return projectId, fmt.Errorf("no project matching the given projectName %s was found", projectName)
```

Comment:
How about?

return "", fmt.Errorf("no project matching the given projectName %s was found", projectName)